helpers/build.js

```diff
@@ -0,0 +1,17 @@
+const uglify = require('uglify-es');
+const fs = require('fs');
+
+const scopeStart = '(function(){';
+const scopeEnd = `})();`;
+
+module.exports.buildFile = (fileName) => {
+
+    let codeString = fs.readFileSync(`./${fileName}.js`, 'utf-8');
+    codeString = `${scopeStart} ${codeString} ${scopeEnd}`;
+
+    const result = uglify.minify(codeString);
```

Comment:
Don't we want this to be fully uglified? By default this just does minification.